api/pom.xml

```diff
@@ -27,35 +27,29 @@
 
     <groupId>edu.wgu.osmt</groupId>
     <artifactId>osmt-api</artifactId>
-    <version>2.6.0-SNAPSHOT</version>
+    <version>2.6.1-SNAPSHOT</version>
```

Comment:
Please don't make any changes to Maven version values. Maven will do this for us via release plugin.